packages/sdk/src/components/sdk-get/sdk-get.tsx

```diff
@@ -40,6 +40,14 @@ type OwnProps = {
 export type Props = DispatchProps & OwnProps;
 type StaticErrorHandler = (error: TSdkError) => void;
 
+const defaultProps: Required<
+  Pick<Props, 'actionCreatorArgs' | 'shouldRefetch'>
+> = {
+  actionCreatorArgs: [],
+  shouldRefetch: (prevArgs: TActionCreatorArgs, nextArgs: TActionCreatorArgs) =>
+    !deepEqual(prevArgs, nextArgs),
+};
```

Comment:
Just creating an object with the default props (nothing to do with React's default props).